sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1421,3 +1423,14 @@ def _more_tags(self):
     # to exist so that third party estimators can easily support multiple sklearn
     # versions.
     check_estimator_tags_renamed("OkayEstimator", OkayEstimator())
+
+
+# Test that set_output doesn't make the tests to fail.
+@parametrize_with_checks(
+    [
+        StandardScaler().set_output(transform="pandas"),
+        StandardScaler().set_output(transform="polars"),
```

Comment:
You might need to have some `skipif` because polars or pandas are optional dependencies.

Reply:
these lines don't fail if pandas/polars are not installed, and `check_estimator` also dosn't fail when they're not installed.

Reply:
Scratch that, it fails. I'll fix it.

Reply:
:P